bugbot/rules/webcompat_sightline.py

```diff
@@ -0,0 +1,76 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from typing import Any, Optional
+
+from bugbot import gcp
+from bugbot.bzcleaner import BzCleaner
+
+
+class WebcompatSightline(BzCleaner):
+    WHITEBOARD_ENTRY = "[webcompat:sightline]"
+
+    def __init__(self):
+        super().__init__()
+        self.sightline_ids = set()
+
+    def description(self) -> str:
+        return "Web Compat site report in the sightline metric set"
```

Comment:
The description will be used as the title of the email.
```suggestion
        return "Bugs with the [webcompat:sightline] whiteboard tag updated"
```